### ISSUE TITLE
RFE: Enhanced Dashboard

### ISSUE THREAD

**Issue by msilmser** [ansible/awx#1973]
##### ISSUE TYPE
 - Feature Idea

##### COMPONENT NAME
- UI

##### SUMMARY
In Tower's dashboard, the "Recently Used Templates" and "Recent Run Jobs" displays are not informative enough to be useful indicators of how well Tower is functioning.  The main problem is that the display is binary -- green or red only -- and that a job run against 150 hosts with a single host failure looks the same as one where every host fails.  

##### ADDITIONAL INFORMATION

Please consider refactoring the dashboard to use more than just a single dot to display status.  For example:
1) Enlarge the dots so that they can include numbers for succeeded/failed hosts (as in "149/1" or "0/150" in the hypotheticals above).  Alternately, add those numbers in the flyover text already provided. 
2) Use a wider range of colors that gives a more useful at-a-glance indicator of job effectiveness.  For example, green = 95% of hosts were successful; yellow/orange = 50-95% success; red = under 50% success.  You could even use gradients in that middle range so the orange is darker at 55% than at 80%.

These UI tweaks would make the dashboard displays more useful and less alarming to look at.

**Comment by AlanCoding** [comment]
> The main problem is that the display is binary -- green or red only -- and that a job run against 150 hosts with a single host failure looks the same as one where every host fails.

Yes, and a single host can have both failed and successful tasks inside of a playbook. So there are really 3 levels of granularity:

 - job status
 - host status
 - task status

If you look at the `JobHostSummary` model, it contains the information for the latter 2 here. It would be relatively trivial to provide this information on a by-host basis or a by-task basis as opposed to a by-job basis in the code where we produce the counts:

https://github.com/ansible/awx/blob/4fe7c9ea427f5b00cd89a6a1583a7bf9c5eb9a61/awx/api/views.py#L601-L603

I don't want to dive into the much more complex full-stack changes that would be needed to display both kinds information on the same graph.

Going back to that time-series graph, you could replace some part of it with:

```python
qsstats.QuerySetStats(JobHostSummary.objects.filter(failed=True), 'created').time_series(end_date, start_date, interval='days')
```

This might not be 100% accurate data, but it would get the gist of what you're going for.

**Comment by unlikelyzero** [comment]
@msilmser have you had a chance to take a look at the new dashboard?

**Comment by msilmser** [comment]
@unlikelyzero i have not, ill spin up an awx instance today and check it out.

**Comment by msilmser** [comment]
@unlikelyzero i have an awx spun up but im not seeing the specific customer asks. maybe I missed something (very likely)?

**Comment by wenottingham** [comment]
There's not significant changes here in the new UI yet.
